### PR TITLE
Refresh images to reflect new UX

### PR DESCRIPTION
**Pull request by urischeiner**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by urischeiner** [comment]
Refresh images + additional doc on Provision and Destroy pipeline

File: docs/infra-as-code-management/workspaces/create-workspace.md [hunk 1]
```diff
@@ -43,15 +43,16 @@ To create a new workspace, follow these steps:
...
 
         ![Add workspace details](./static/new-workspace.png)
 
 6. Select **Save**. 
 
-7. If you need to use either environment or Terraform variables during execution, select the **Variables** tab to define the variables. If Terraform variables are defined and set in Git, then values defined in the workspace take precedence. 
-  A good best practice is to avoid managing Terraform variables in Git, and set them in the workspace instead. This keeps the Git configuration clean and allows you to use the same Git definitions for multiple workspaces. 
+7. If you need to use either environment or Terraform variables during execution, select the **Variables** tab to define the variables. If Terraform variables are defined and set in Git, then values defined in the workspace take precedence.
+Variables can be either String, [hcl](https://developer.hashicorp.com/terraform/language/syntax/configuration), [Secret](https://developer.harness.io/docs/category/secrets), or a reference to another variable in the same or a different Workspace.
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/create-workspace.md:54]:
```suggestion
Variables can be either String, [hcl](https://developer.hashicorp.com/terraform/language/syntax/configuration), [Secret](https://developer.harness.io/docs/category/secrets), or a reference to another variable in the same or a different workspace.
```

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 2]
```diff
@@ -28,23 +28,23 @@ import TabItem from '@theme/TabItem';
 
 4. Enter a stage name and an optional description and tag. Select **Set Up Stage**. 
 
-5. On the **Workspace** tab, select an existing workspace or select **+ Create New Workspace** to create a new workspace. 
+5. On the **Workspace** tab, select an existing workspace or select **+ Create New Workspace** to create a new Workspace. A workspace can also be determined during execution if the value is set to be runtime input
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:31]:
```suggestion
5. On the **Workspace** tab, select an existing workspace or select **+ Create New Workspace** to create a new workspace. A workspace can also be determined during execution if the value is set to be `runtime input`.
```

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 3]
```diff
@@ -28,23 +28,23 @@ import TabItem from '@theme/TabItem';
...
 
-6. Select the **Execution** tab, and then select the option to provision for Terraform. Five operations are supported: init, plan, apply, destroy, plan-destroy.
+6. Select the **Execution** tab, and then select "Provision" from 'Common Operations'.
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:33]:
```suggestion
6. Select the **Execution** tab, and then from **Common Operations**, select **Provision**.
```

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 4]
```diff
@@ -28,23 +28,23 @@ import TabItem from '@theme/TabItem';
...
 
-7. Select **Use Strategy**.
+7. Click **Use Strategy**.
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:35]:
```suggestion
7. Select **Use Strategy**.
```

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 5]
```diff
@@ -106,40 +106,30 @@ pipeline:
...
-
-* Lastly, the destroy step destroys the plan generated in the previous step.
 
 ```mdx-code-block
 <Tabs>
   <TabItem value="Visual" label="Visual">
 ```
 
-1. Select **Add Step**.
+1. Follow Steps 1-5 as described in the "Provision a Workspace" section
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:118]: Don't make users go hunt for steps. Provide the steps here.

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 6]
```diff
@@ -106,40 +106,30 @@ pipeline:
...
+2. Select the **Execution** tab and select "Destroy" from 'Common Operations'.
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:119]:
```suggestion
2. On the **Execution** tab, in **Common Operations**, select **Destroy**.
```

File: docs/infra-as-code-management/workspaces/provision-workspace.md [hunk 7]
```diff
@@ -106,40 +106,30 @@ pipeline:
...
+3. Click **Use Strategy**.
```
Comment (KimberlyFields) [inline docs/infra-as-code-management/workspaces/provision-workspace.md:120]:
```suggestion
3. Select **Use Strategy**.
```

File: docs/infra-as-code-management/workspaces/create-workspace.md (hunk 1 above)
Comment (urischeiner) [reply docs/infra-as-code-management/workspaces/create-workspace.md:54]: @KimberlyFields Workspace is an entity (like Service) - shouldn't we use capital?

Comment (KimberlyFields) [reply docs/infra-as-code-management/workspaces/create-workspace.md:54]: @urischeiner No. It's not a proper noun. Unless we have trademarked Harness Workspace, then it should not be capitalized.

Comment (urischeiner) [reply docs/infra-as-code-management/workspaces/create-workspace.md:54]: sounds good, you can then merge the PR if you are ok

**Comment by urischeiner** [comment]
I think we're good. I will make the changes for the steps in the next PR

**Review by KimberlyFields — APPROVED** [review]

**Pull request merged**